### DISCUSSION TITLE
Projeto portfólio

### DISCUSSION THREAD

**Discussion by jesielviana** [ifpi-picos/ads-programacao-para-internet-i#27] [General]
Olá devs, 

Gostaria de solicitar que vocês respondessem  esta discussão com o link para o repositório do projeto de portfólio de vocês.

**Comment by ghost** [comment]
https://github.com/RodrigoPereira25/rodrigopereira25.github.io.git - Portfólio │ Rodrigo Pereira

**Comment by Caio-Leto** [comment]
https://github.com/Caio-Leto/Portifolio - Portifólio | Caio Vinícius

**Comment by Allan-Barbosa** [comment]
https://github.com/Allan-Barbosa/Allan-Barbosa.github.io - Portfólio | Allan Barbosa

**Comment by alanfilho184** [comment]
https://github.com/alanfilho184/Kami-Site - Projeto para portfólio | Alan Leal

**Comment by Rafhael0069** [comment]
https://rafhael0069.github.io/portfolio/ portfólio - Rafhael

**Reply by jesielviana** [reply]
Perfeito, Rafhael. Ficou muito bom.

**Reply by Rafhael0069** [reply]
Tem só três coisas que eu que eu quero tirar duvidas com o professor, mas só na próxima aula.

**Comment by ezequiasmoura** [comment]
https://github.com/ezequias0203/ezequias0203.gihub.io.git

**Reply by jesielviana** [reply]
Oá @ezequias0203, está faltando o link do site online.

**Reply by ezequiasmoura** [reply]
https://ezequias0203.github.io/ezequias0203.gihub.io/

**Reply by jesielviana** [reply]
Ok, você deve renomear o nome do repositório pro site ficar na raiz do seu usuário. Seu repositório deve ter o nome "ezequias0203.github.io". No atual está "ezequias0203.gihub.io" (está faltando o "t" no "github").

**Reply by ezequiasmoura** [reply]
Ok

**Reply by ezequiasmoura** [reply]
https://ezequias0203.github.io/

**Reply by jesielviana** [reply]
perfeito!

**Comment by Italopax** [comment]
https://github.com/Italopax/Italopax.github.io - Portifólio | Italo Paixão Gomes

**Comment by giselaAS** [comment]
https://github.com/giselaAS/giselaAS.github.io - Portifolio | Gisela

**Comment by victorsaless** [comment]
https://github.com/victorsaless/victorsaless.github.io - portfólio | Sales

**Comment by WesleyVeloso30** [comment]
https://github.com/WesleyVeloso30/wesleyveloso30.github.io  - portfolio / Wesley

**Comment by DiesonSDB** [comment]
https://diesonsdb.github.io  - portfólio / Dieson

**Comment by Anth0nYM** [comment]
> Olá devs,
> 
> Gostaria de solicitar que vocês respondessem esta discussão com o link para o repositório do projeto de portfólio de vocês.
--------------------------------------------------------------------------------------------------------------------------------------------------------
Seguem respectivamente os links em anexo do repositório e do site:
https://github.com/Anth0nYM/Portfolio
https://anth0nym.github.io/Portfolio/

Outro site:
https://github.com/Anth0nYM/Anth0nYM.github.io
https://anth0nym.github.io/

**Reply by jesielviana** [reply]
show!

**Comment by IagoBrenoAD** [comment]
https://github.com/IagoBrenoAD/IagoBrenoAD.github.io - Portfólio | Iago

**Comment by AlessandroVianaSilva** [comment]
https://github.com/AlessandroVianaSilva/AlessandroVianaSilva.github.io - Portfolio | Alessandro

**Comment by marcielysilva** [comment]
https://github.com/marcielysilva/marcielysilva.github.io | MARCIELY FERREIRA
